ClothesApp/WebAPI/Controllers/OrderController.cs

```diff
@@ -0,0 +1,81 @@
+using Application.Dtos.OrderItems;
+using Application.Dtos.Orders;
+using Application.Dtos.OrderTransactions;
+using Application.Interfaces.Services;
+using Domain.Entities;
+using Domain.Enums;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[Route("api/orders")] 
+public class OrderController : ControllerBase
```

Comment:
How would you return orders to admin to make some actions? How to know what orders are pending, etc...?